dev/com.ibm.websphere.appserver.spi.ssl/bnd.bnd

```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017 IBM Corporation and others.
+# Copyright (c) 2019 IBM Corporation and others.
```

Comment:
Please change this to `2017, 2019` ... years of first and last publication

Reply:
Changes have been made in the last push.